### PR TITLE
infra: verify behavior of check NPE in CI

### PR DESCRIPTION
**Pull request by nmancus1**
This PR will verify that exception in a check will fail CI jobs.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 1]
```diff
@@ -160,6 +160,9 @@
 
     @Override
     public void visitToken(DetailAST ast) {
+        if (ast != null) {
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:163]: Don't know whats going on, but it should be impossible for a `visitToken` to give a `null` AST.

**Comment by romani** [comment]
@nmancus1 , we have problem
https://travis-ci.org/github/checkstyle/checkstyle/builds/769223527 , "no-exception" jobs are not failing.

**Comment by rnveach** [comment]
@romani I think this may be related to https://github.com/checkstyle/checkstyle/pull/9972#issuecomment-830827761 .
We should recheck all our launch to diff conversion.

**Comment by nmancus1** [comment]
All no exception testing with Java checks failed as expected; no exception tests that did not fail were Javadoc checks. I am closing this PR, we have verified that diff.groovy behavior is correct.

**Pull request closed without merging**